src/uct/tcp/tcp_iface.c

```diff
@@ -111,6 +111,12 @@ static ucs_config_field_t uct_tcp_iface_config_table[] = {
                 UCS_CONFIG_TYPE_TIME_UNITS},
 #endif /* UCT_TCP_EP_KEEPALIVE */
 
+  {"EP_BIND_SRC_ADDR", "y",
+   "Bind internal socket to the interface before connection establishment to"
```

Comment:
"Bind client socket to the local network interface before connecting to the remote peer"